src/logic/FetchRemotes.ts

```diff
@@ -130,8 +145,8 @@ export async function fetchExtensionManifest(contents_url: string, branch: strin
 
     return parsedManifests;
   }
-  catch (err) {
-    // console.warn(contents_url, err);
+  catch (err: any) {
```

Comment:
Types

Reply:
You can only assign either unknown or any to caught errors and type check is done in the catch block anyway.